### ISSUE TITLE
Need a new URL for downloading `wkhtmltopdf`

### ISSUE THREAD

**Issue by smondet** [hammerlab/secotrec#63]
`gna.org` has closed: https://en.wikipedia.org/wiki/Gna%21

Cf.
https://github.com/hammerlab/secotrec/blob/master/src/app/make_dockerfiles.ml#L186
it was copied from Biokepi:
https://github.com/hammerlab/biokepi/pull/386

Cc: @armish @julia326 @iskandr

**Comment by julia326** [comment]
@smondet thanks for catching this, this link should work: http://downloads.wkhtmltopdf.org/0.12/0.12.3/wkhtmltox-0.12.3_linux-generic-amd64.tar.xz

**Comment by smondet** [comment]
@julia326 that link also times-out for me (and for docker-build on gcloud)?

**Comment by julia326** [comment]
@smondet not sure why, this works for me:

```
julia@fuzzy:~$ wget http://downloads.wkhtmltopdf.org/0.12/0.12.3/wkhtmltox-0.12.3_linux-generic-amd64.tar.xz
--2017-06-05 20:19:03--  http://downloads.wkhtmltopdf.org/0.12/0.12.3/wkhtmltox-0.12.3_linux-generic-amd64.tar.xz
Resolving downloads.wkhtmltopdf.org (downloads.wkhtmltopdf.org)... 137.74.141.30
Connecting to downloads.wkhtmltopdf.org (downloads.wkhtmltopdf.org)|137.74.141.30|:80... connected.
HTTP request sent, awaiting response... 301 Moved Permanently
Location: https://downloads.wkhtmltopdf.org/0.12/0.12.3/wkhtmltox-0.12.3_linux-generic-amd64.tar.xz [following]
--2017-06-05 20:19:04--  https://downloads.wkhtmltopdf.org/0.12/0.12.3/wkhtmltox-0.12.3_linux-generic-amd64.tar.xz
Connecting to downloads.wkhtmltopdf.org (downloads.wkhtmltopdf.org)|137.74.141.30|:443... connected.
HTTP request sent, awaiting response... 200 OK
Length: 14531204 (14M) [application/octet-stream]
Saving to: ‘wkhtmltox-0.12.3_linux-generic-amd64.tar.xz’

wkhtmltox-0.12.3_linux-generic-amd64.ta 100%[===============================================================================>]  13.86M  8.08MB/s   in 1.7s   
```
in what situation is it timing out for you?

**Comment by smondet** [comment]
Ah yes, now it works
(`http://downforeveryoneorjustme.com` also said it was down before).
Thanks!

**Comment by armish** [comment]
Looks like this is available through conda so we can just add this as a `basepackage` when installing `vaxrank` and I think that will save you from quite a lot of trouble, @smondet, right?

@julia326: do you only use this in vaxrank or do we have other tools that depend on this?

**Comment by smondet** [comment]
@armish getting it with biokepi would make life easier for people trying to use Biokepi in a docker-free environment.

**Comment by smondet** [comment]
Fixed by #65.

**Comment by julia326** [comment]
@armish as far as I know it's just vaxrank - though the other issue with installing with conda is that you need to grab the source rather than the package itself, otherwise the output is weirdly formatted. I don't remember the specifics for why that happens but could dig those up if you're interested.
